sql/src/test/java/io/crate/analyze/DropFunctionAnalyzerTest.java

```diff
@@ -45,6 +51,42 @@ public void testDropFunctionSimple() throws Exception {
         assertThat(analyzedStatement, instanceOf(DropFunctionAnalyzedStatement.class));
 
         DropFunctionAnalyzedStatement analysis = (DropFunctionAnalyzedStatement) analyzedStatement;
+        assertThat(analysis.schema(), is("doc"));
+        assertThat(analysis.name(), is("bar"));
+        assertThat(analysis.ifExists(), is(false));
+        assertThat(analysis.argumentTypes().get(0), is(DataTypes.LONG));
+        assertThat(analysis.argumentTypes().get(1), is(DataTypes.OBJECT));
+    }
+
+    @Test
+    public void testDropFunctionWithSessionSetSchema() throws Exception {
+        DropFunctionAnalyzedStatement analysis = (DropFunctionAnalyzedStatement) e.analyzer.boundAnalyze(
+            SqlParser.createStatement("DROP FUNCTION bar(long, object)"),
+            new SessionContext(0, Option.NONE, "my_schema"),
+            new ParameterContext(Row.EMPTY, Collections.emptyList())).analyzedStatement();
+
+        assertThat(analysis.schema(), is("my_schema"));
+        assertThat(analysis.name(), is("bar"));
+    }
+
+    @Test
+    public void testDropFunctionWithExplicitSchema() throws Exception {
+        DropFunctionAnalyzedStatement analysis = (DropFunctionAnalyzedStatement) e.analyzer.boundAnalyze(
+            SqlParser.createStatement("DROP FUNCTION my_other_schema.bar(long, object)"),
+            new SessionContext(0, Option.NONE, "my_schema"),
+            new ParameterContext(Row.EMPTY, Collections.emptyList())).analyzedStatement();
+
+        assertThat(analysis.schema(), is("my_other_schema"));
+        assertThat(analysis.name(), is("bar"));
+    }
+
+    @Test
+    public void testDropFunctionSchema() throws Exception {
```

Comment:
This test is not needed, everything is already tested here:
https://github.com/crate/crate/pull/5225/files#diff-334a5b80aaf90336a30f754b6a031413R54
and here:
https://github.com/crate/crate/pull/5225/files#diff-334a5b80aaf90336a30f754b6a031413R73